kolibri/utils/options.py

```diff
@@ -737,6 +766,7 @@ def _get_validator():
             "url_prefix": url_prefix,
             "bytes": validate_bytes,
             "multiprocess_bool": multiprocess_bool,
+            "storage_option": storage_option,
```

Comment:
how will this work with database based cache?

Reply:
Hm - I'm not sure. I assumed this was only related to validation on initialization